src/Makefile.am

```diff
@@ -24,7 +24,7 @@ CARGO_FEATURES += --features tracy
 endif
 
 if ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
-CARGO_FEATURES += --features core-vnext
+CARGO_FEATURES += --features core-vnext --features soroban-env-host-prev
```

Comment:
We don't actually want to merge this now because we want the vnext runs to always use the vnext env.